speaker/layer2_controller.go

```diff
@@ -37,6 +37,16 @@ func (c *layer2Controller) SetConfig(log.Logger, *config.Config) error {
 	return nil
 }
 
+func activeNodes(mList *memberlist.Memberlist) []string {
```

Comment:
Could you please run `go fmt` on this file? It has inconsistent indentation.

Reply:
I've raised #632 to track adding a CI job to catch this sort of thing automatically